pkg/endpoint/policy.go

```diff
@@ -590,12 +592,19 @@ func (e *Endpoint) regeneratePolicy(owner Owner, opts models.ConfigurationMap) (
 
 	needToRegenerateBPF := optsChanged || policyChanged || e.nextPolicyRevision > e.policyRevision
 
+	regenerateEnd := time.Since(regenerateStart) / time.Second
+	e.getLogger().WithField(logfields.PolicyRegenerationTime, regenerateEnd).
+		Info("Regeneration of policy has completed")
+	metrics.PolicyRegenerationTime.Add(float64(regenerateEnd))
```

Comment:
Also increment a counter of policy regenerations.

Reply:
We can already derive that value using` policy_count` i.e `Number of policies currently loaded` and `policy_import_errors` i.e `Number of times a policy import has failed`. I felt adding a third counter would be redundant.

Reply:
let me know your thoughts.